<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1188</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="styleSheet" native="true">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>951</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	margin: 0;
	padding: 0;
}

QPushbutton{background-color: green;}
QPushbutton:hover{background-color: yellow;}
QPushbutton:pressed{background-color: red;}</string>
   </property>
   <widget class="QFrame" name="app">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>911</width>
      <height>531</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,6">
     <item>
      <widget class="QFrame" name="LeftMenu">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="btn1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>home</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>btn2</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>290</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>btn3</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="pages">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>641</width>
          <height>421</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="page1">
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>-10</y>
            <width>721</width>
            <height>441</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>C:/Users/Administrator/Desktop/images/background.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page2">
         <widget class="QComboBox" name="comboBox">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>70</y>
            <width>69</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QTextEdit" name="textEdit">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>70</y>
            <width>471</width>
            <height>311</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page3">
         <widget class="QCalendarWidget" name="calendarWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>631</width>
            <height>401</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
